Sources/PaymentsCore/GetClientIdRequest.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+struct GetClientIdRequest: APIRequest {
+
+    typealias ResponseType = GetClientIdResponse
+
+    let path: String = "v1/oauth2/token"
+    let token: String
+    var method: HTTPMethod = .get
+    var body: Data?
+
+    var headers: [HTTPHeader: String] {
+        return [
```

Comment:
Can remove the return keyword and rely on implicit return

Reply:
Should be all good!